lib/librato/rails/configuration.rb

```diff
@@ -46,14 +49,30 @@ def configure_with_config_file
         env_specific = YAML.load(ERB.new(File.read(config_file)).result)[::Rails.env]
         if env_specific
           settable = CONFIG_SETTABLE & env_specific.keys
-          settable.each { |key| self.send("#{key}=", env_specific[key]) }
+          settable.each do |key|
+            value = env_specific[key]
+            value.symbolize_keys! if key == "tags"
+            self.send("#{key}=", value)
+          end
         end
       end
 
       def default_suites
         super + DEFAULT_SUITES
       end
 
+      def default_tags
+        {
+          service: ::Rails.application.class.to_s.split("::").first.underscore,
```

Comment:
[`#deconstantize`](http://apidock.com/rails/v3.2.1/String/deconstantize) is unfortunately not available for rails 3.1